client/landing/gutenboarding/components/link/index.tsx

```diff
@@ -46,7 +45,7 @@ type Assign3< T extends object, U extends object, V extends object > = Assign< T
 // LinkProps _without_ component
 // LinkProps `to` is optional
 type Props = Omit<
-	Assign3< Button.AnchorProps, Omit< LinkProps, 'component' >, Partial< Pick< LinkProps, 'to' > > >,
+	Assign3< Button.AnchorProps, Omit< LinkProps, 'component' >, { to?: string } >,
```

Comment:
Since we've reverted all the other changes in this file, I think we should also revert this one :sweat_smile:

Reply:
Fixed. https://github.com/Automattic/wp-calypso/pull/39312/commits/f3ca829a3730ef2832a1a240aca921683b63ae59